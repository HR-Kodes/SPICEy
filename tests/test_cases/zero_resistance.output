Error: R2 has zero resistance.
