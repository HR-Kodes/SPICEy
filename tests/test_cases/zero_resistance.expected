{
  "error_code": 4
}